src/Asset.js

```diff
@@ -65,6 +65,10 @@ class Asset {
       return url;
     }
 
+    if (url.indexOf('/') === 0) {
+      url = path.join(this.options.rootDir, url);
```

Comment:
This will work for files that are in the direct project being bundled, but what about absolute paths inside packages in `node_modules`? Seems like those should probably be relative to that package, not the whole project maybe?

Reply:
Ow didn't know that happened i'll try find a better way than